include/proxy-wasm/sdk.h

```diff
@@ -0,0 +1,54 @@
+// Copyright 2016-2019 Envoy Project Authors
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <string>
+#include <cstdint>
+
+namespace proxy_wasm {
+
+namespace internal {
+
+// isolate those includes to prevent ::proxy_wasm namespace pollution with std
+// namespace definitions.
+namespace proxy_wasm {
```

Comment:
I will try to respond today or tomorrow

Reply:
I think we can easily remove it.
I thought that it would be nice to see in the `using` statement the source of the coming type-id.
By having `internal::proxy_wasm` it is easy to understand that those aliases are coming from the proxy_wasm library.
I can remove it if you want.
WDYT?

Reply:
Was this discussion resolved offline?

Reply:
@PiotrSikora no, it is waiting for @mpwarres input

Reply:
Odd, I thought I had added a further comment here, but apparently I forgot to send it.

Since the `using` statements are in proxy-wasm/sdk.h itself, personally I think that `using WasmResult = internal::WasmResult` is clear enough, and avoids the redundancy in `proxy_wasm::internal::proxy_wasm::WasmResult`.

However, since (per @yanavlasov comment) this is blocking Envoy toolchain imports, I am ok with submitting as-is.

Reply:
Ok, let me know if you want me to rework it to use WasmResult = internal::WasmResult.

Reply:
Yes, if you don't mind, can you change it to that? Thanks.

Reply:
Yeah, totally, I will patch today!